### PR TITLE
Add skip button in wizard

### LINKED ISSUE #345: There should be a Skip button on the Onboarding Import section [Feature request]
### Description

Hey, I am making this feature request after receiving feedback from many different users on the review section of the plugin on the wp.org forum and they are not satisfied with the changes that we have introduced in the plugin after version 2.4.7.

The changes that were introduced :

1. Adds compatibility with Block Editor or any page builder for building the maintenance mode page.
2. Adds coming soon and maintenance mode starting templates.
3. Installing the Otter plugin after the import process.

However, some users have liked the above enhancement and gave us a very positive review on the forum and have started using the Otter plugin as well but some users feel that they are being forced to install another plugin in order to make the LightStart Plugin (the New rebranded name) works for them and have suggested that there should be a skip button on the Start page where the users are asked to choose a template. This should skip the process and directly redirect them to the settings page of the plugin.

### Reference
Wp.org review section: https://wordpress.org/support/plugin/wp-maintenance-mode/reviews/

### PR DESCRIPTION
**Pull request by arinaturcu**
## Summary
- Adds a skip button in the wizard to let the user skip importing a template and installing Otter
- Adds a notice in the wizard and in the design tab to announce that, by importing a template, Otter will be installed and activated (only if Otter is not active already).
- The text in the skip button changes depending on the selected template type (maintenance / coming soon / landing page)

## Screenshots
Screenshot 2022-12-14 at 00 11 03
Screenshot 2022-12-14 at 00 12 01

## Test instructions
- By clicking the `I don’t want to use a template` button from the wizard, the user is taken to the next step, which is the Subscribe step, and then directly to the settings page (skipping the last usual step)
- After this, Otter should not be added to the plugins list

Closes #345.

### CODE REVIEW

**Review by HardeepAsrani — CHANGES_REQUESTED** [review]
I see the Otter message appears even if Otter is already active. We don't need to show it if Otter already exists.

**Comment by arinaturcu** [comment]
Thanks @HardeepAsrani! I've changed it now it only shows if Otter is not active.

File: views/settings.php [hunk 1]
```diff
@@ -44,7 +44,12 @@
 					<div class="step-wrap">
 						<div class="step import-step">
 							<h4 class="header"><?php esc_html_e( 'Get started with a free template.', 'wp-maintenance-mode' ); ?></h4>
-							<p class="description"><?php esc_html_e( 'Just click on one of the pre-designed templates below to get started. You can always edit and customize later, so these are a perfect starting point!', 'wp-maintenance-mode' ); ?></p>
+							<p class="description"><?php esc_html_e( 'Choose the type of template you want, powered by Otter Blocks. You can always customise your layout.', 'wp-maintenance-mode' ); ?></p>
+							<?php
+							if ( ! is_plugin_active( 'otter-blocks/otter-blocks.php' ) ) {
```
Comment (HardeepAsrani) [inline views/settings.php:49]: It's not the best way to detect as someone might have Otter with a different folder name, you can also maybe check this *OR* `OTTER_BLOCKS_VERSION` constant.

Comment (HardeepAsrani) [reply views/settings.php:49]: After that you can put it for QA.

**Comment by irinelenache** [comment]
@arinaturcu Tested the enhancement and here is what i found:

- [ ] If Otter is not installed, the buttons will be at the [bottom of the screen](https://vertis.d.pr/i/mLCm5t), maybe we can add some bottom margin there

**Comment by arinaturcu** [comment]
Thank you @irinelenache!
It's fixed now, let me now if there is anything else.

**Comment by irinelenache** [comment]
@arinaturcu Everything's fine now, thank you 🚀

**Pull request merged**